tests/Faithlife.Data.Tests/DataRecordExtensionsTests.cs

```diff
@@ -513,12 +560,18 @@ private class ItemRecord
 			public byte[]? TheBlob { get; set; }
 		}
 
+#if NET5_0
+#pragma warning disable CA1801,SA1313
+		private record ItemRecord(string? TheString, int TheInt32, long TheInt64, bool TheBool, float TheSingle, double TheDouble, byte[]? TheBlob, int TheOptionalInt32 = 42);
```

Comment:
Can we actually test the behavior by moving `TheInt32` to the end and giving it a default value?

Wouldn't hurt to make a parallel change to `ItemDto`: `public int TheInt32 { get; set; } = 42;`

Reply:
What behavior do you want to test? The result from a data record overwriting a default value?

Reply:
The behavior of the data record not overwriting the default value, e.g. test line 337. It's good to have a test that fails if the code is deleted.

Reply:
The test does fail if https://github.com/Faithlife/FaithlifeData/pull/19/files#diff-821e783579030f09e8cbb23df0e99c71e34806dbbd68790991d77b132b6baa8dR117-R118 is deleted. Am I misunderstanding?

Reply:
No, I guess I was. Carry on!